en/admin/index.rst

```diff
@@ -8,7 +8,7 @@ CUBRID Management
 
 This chapter describes how the database administrators (**DBA**) operates the CUBRID system.
 
-*   It includes instructions on how to use the **cubrid** utility, which starts and stops various processes of the CUBRID server, the broker and manager server. See :doc:`/admin/control`.
+*   It includes instructions on how to use the **cubrid** utility, which starts and stops various processes of the CUBRID server, the broker, java stored procedure server and manager server. See :doc:`/admin/control`.
```

Comment:
the CUBRID server, the broker, java stored procedure server and manager server
-->
the CUBRID server, the broker, java stored procedure server, and manager server
(comma missed)

Reply:
Thank you, I'll fix it